From adf3dd4e8d1c62a5013683ffd15ab1be45970850 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Fri, 11 Mar 2022 11:01:18 -0800
Subject: Cleanup stale protobuf gen stuff (#37861)

Accidentally created a top level file; its not used. Also the old
operator proto is now dead code, removed
---
 buf.gen.yaml         |  8 --------
 operator/operator.mk | 38 --------------------------------------
 2 files changed, 46 deletions(-)
 delete mode 100644 buf.gen.yaml
 delete mode 100644 operator/operator.mk

diff --git a/buf.gen.yaml b/buf.gen.yaml
deleted file mode 100644
index 8c5d7b16a5..0000000000
--- a/buf.gen.yaml
+++ /dev/null
@@ -1,8 +0,0 @@
-# buf.gen.yaml sets up the generation configuration for all of our plugins.
-# Note: buf does not allow multi roots that are within each other; as a result, the common-protos folders are
-# symlinked into the top level directory.
-version: v1beta1
-plugins:
-- name: gogofast
-  out: .
-  opt: plugins=grpc,paths=source_relative,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto,Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,Mgoogle/rpc/status.proto=istio.io/gogo-genproto/googleapis/google/rpc,Mgoogle/rpc/code.proto=istio.io/gogo-genproto/googleapis/google/rpc,Mgoogle/rpc/error_details.proto=istio.io/gogo-genproto/googleapis/google/rpc,Mgoogle/api/field_behavior.proto=istio.io/gogo-genproto/googleapis/google/api
diff --git a/operator/operator.mk b/operator/operator.mk
deleted file mode 100644
index cb9f94403d..0000000000
--- a/operator/operator.mk
+++ /dev/null
@@ -1,38 +0,0 @@
-# Copyright Istio Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-pwd := $(shell pwd)
-
-TMPDIR := $(shell mktemp -d)
-
-repo_dir := .
-out_path = ${TMPDIR}
-protoc = protoc -Icommon-protos -Ioperator
-
-go_plugin_prefix := --gogofast_out=plugins=grpc,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types
-go_plugin := $(go_plugin_prefix):$(out_path)
-
-v1alpha1_path := operator/pkg/apis/istio/v1alpha1
-v1alpha1_protos := $(wildcard $(v1alpha1_path)/*.proto)
-v1alpha1_pb_gos := $(v1alpha1_protos:.proto=.pb.go)
-v1alpha1_openapi := $(v1alpha1_protos:.proto=.json)
-
-$(v1alpha1_pb_gos): $(v1alpha1_protos)
-	@$(protoc) $(go_plugin) $^
-	@cp -r ${TMPDIR}/pkg/* operator/pkg/
-	@rm -fr ${TMPDIR}/pkg
-	@go run $(repo_dir)/operator/pkg/apis/istio/fixup_structs/main.go -f $(v1alpha1_path)/values_types.pb.go
-
-#.PHONY: operator-proto $(v1alpha1_pb_gos)
-#operator-proto: $(v1alpha1_pb_gos)
-- 
2.35.3

